metadata-ingestion/src/datahub/ingestion/source/powerbi/config.py

```diff
@@ -130,21 +148,27 @@ class PowerBiAPIConfig(EnvBasedSourceConfigBase):
     )
     # Enable/Disable extracting ownership information of Dashboard
     extract_ownership: bool = pydantic.Field(
-        default=True, description="Whether ownership should be ingested"
+        default=True,
+        description="Whether ownership should be ingested. Admin access should be given to the configured "
+        "Azure AD Application ",
     )
     # Enable/Disable extracting report information
     extract_reports: bool = pydantic.Field(
         default=True, description="Whether reports should be ingested"
     )
     # Enable/Disable extracting lineage information of PowerBI Dataset
     extract_lineage: bool = pydantic.Field(
-        default=True, description="Whether lineage should be ingested"
+        default=True,
+        description="Whether lineage should be ingested. Admin access should be given to the configured Azure AD "
+        "Application "
+        "should be set to 'true' ",
     )
     # Enable/Disable extracting endorsements to tags. Please notice this may overwrite
-    # any existing tags defined to those entitiies
+    # any existing tags defined to those entities
     extract_endorsements_to_tags: bool = pydantic.Field(
         default=False,
-        description="Whether to extract endorsements to tags, note that this may overwrite existing tags",
+        description="Whether to extract endorsements to tags, note that this may overwrite existing tags. Admin "
```

Comment:
Same as above: "Admin API access is required is this setting is enabled"

Reply:
done